src/status_im/ui2/screens/chat/actions.cljs

```diff
@@ -117,13 +119,14 @@
           :sub-label nil
           :chevron?  false}))
 
-(defn leave-group-entry [item]
-  (entry {:icon      :i/log-out
-          :label     (i18n/label :t/leave-group)
-          :on-press  #(leave-group-action item)
-          :danger?   true
-          :sub-label nil
-          :chevron?  false}))
+(defn leave-group-entry [item extra-data]
+  (entry {:icon         :i/log-out
+          :label        (i18n/label :t/leave-group)
+          :on-press     #(leave-group-action item (if extra-data (:chat-id extra-data) (:chat-id item)))
+          :danger?      true
+          :sub-label    nil
+          :chevron?     false
+          :add-divider? (when (not= extra-data nil) true)}))
```

Comment:
Can use `some?` to check for `non-nil` vals

Reply:
The whole logic could be replaced by a simpler `(not extra-data)` because we only care about the absence of `extra-data`.

Reply:
Actually can be replaced by just `add-divider?    extra-data`